ccm_web/client/src/components/Help.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { makeStyles, Typography, Link } from '@material-ui/core'
+import { FeatureUIProps } from '../models/FeatureUIData'
+
+const useStyles = makeStyles((theme) => ({
+  helpText: {
+    marginLeft: 'auto',
+    padding: '20px'
+  }
+}))
+
+interface HelpLinkProps {
+  pathnames: string[]
+  features: FeatureUIProps[]
+  baseHelpURL: string
+}
+
+function Help (props: HelpLinkProps): JSX.Element {
+  const classes = useStyles()
+  const { features, pathnames, baseHelpURL } = props
+  let helpURLEnding = baseHelpURL
+  for (const feature of features) {
+    if (`/${pathnames[0]}`.includes(feature.route)) {
+      helpURLEnding += feature.data.helpURLEnding
+    }
+  }
```

Comment:
I think I'd favor moving this searching logic out of the component (this is why I partly why I was interested in the refactor here https://github.com/pushyamig/canvas-course-manager-next/pull/2), but it's not essential I suppose if it works.